Makefile

```diff
@@ -113,9 +119,11 @@ ifeq ($(NO_DOCKER), 1)
 	CGO_ENABLED=0 go build -tags builtinassets -mod=vendor -o ./build/bin/$(BIN_GO_NAME) ./$(BIN_GO_DIR)/$(BIN_GO_NAME)/*.go
 # If pushing, build and tag native arch image to GCR.
 else ifeq ($(DOCKER_PUSH), 1)
-	$(call docker_build, --tag gmp/$(BIN_GO_NAME) -f ./$(BIN_GO_DIR)/$(BIN_GO_NAME)/Dockerfile .)
+	$(call docker_build, --tag "gmp/$(BIN_GO_NAME):$(TAG_NAME)" -f "./$(BIN_GO_DIR)/$(BIN_GO_NAME)/Dockerfile" .)
 	@echo ">> tagging and pushing images"
-	$(call docker_tag_push,gmp/$(BIN_GO_NAME),${IMAGE_REGISTRY}/$(BIN_GO_NAME):${TAG_NAME})
+	$(call docker_tag, "gmp/$(BIN_GO_NAME):$(TAG_NAME)" "gmp/$(BIN_GO_NAME):latest")
+	$(call docker_tag, "gmp/$(BIN_GO_NAME):$(TAG_NAME)" "${IMAGE_REGISTRY}/$(BIN_GO_NAME):$(TAG_NAME)")
```

Comment:
Suggestion: It's possible to do [multiple tags with one build command](https://docs.docker.com/reference/cli/docker/image/build/#tag). No need to call tag separately for this case.

Reply:
That's `docker build`. `docker tag` does not support multiple tags.